source/_patterns/01-molecules/navigation/profile-login-control.mustache

```diff
@@ -0,0 +1,15 @@
+<div class="profile-login-control" {{#isLoggedIn}}data-profile-home-link="{{profileHomeLink}}"
+     data-profile-manage-link="{{profileManageLink}}" data-logout-uri="{{logoutUri}}" data-display-name="{{displayName}}"{{/isLoggedIn}}
```

Comment:
Is there anyway we could decouple what appears in the menu from the template? (So we could add a new user-specific page in Journal without having to change the pattern.)